src/main/java/xyz/vegaone/easytracking/domain/Epic.java

```diff
@@ -0,0 +1,44 @@
+package xyz.vegaone.easytracking.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+import xyz.vegaone.easytracking.utils.Status;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+public class Epic {
+
+    @Id
+    @GeneratedValue
```

Comment:
I don't think this will work with `MySql`. It will need additional information there. See the `User` entity class for reference